index.html

```diff
@@ -146,8 +145,17 @@
     {{ render_tag('framework', game['framework']) }}
 
     {%- if 'info' in game %}<div>{{ game['info'] }}</div>{% endif %}
-    
+
+    {%- set images = [] %}
+    {% for item in game.get('media', []) %}
+      {% if 'image' in item -%}
+        {% do images.append(item['image']) %}
+      {%- endif %}
+    {% endfor -%}
     <div class="gallery" style="display: none"></div>
+    {% if images | length > 0 -%}
+      <script class="gallery-json" type="text/json">{{ images | join(',') }}</script>
```

Comment:
The images are not live DOM though, they're just plain text between `<script>` tags. They're made live when you click the image. So to collect them you'd have to put them in the DOM then read the `src` attribute. The plan is to remove all that and just use the new urls.